Amplitude/Amplitude.m

```diff
@@ -1539,6 +1553,9 @@ - (void)updateLocation
                 _lastKnownLocation = location;
             }
         }
+#else
+        AMPLITUDE_LOG(@"WARNING: AMPLITUDE_LOCATION_TRACKING tracking was disabled");
```

Comment:
My only concern is that `updateLocation` gets called every time the app comes back into foreground. This isn't going to spam logs right? I guess `AMPLITUDE_LOG` is off by default